.github/workflows/integration_tests.yml

```
@@ -0,0 +1,34 @@
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Integration tests
+on: [release, pull_request]
+
+jobs:
+  buildCheck:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt install -y libpopt-dev libselinux1-dev libnewt-dev gettext
+      - name: Build & install
+        run: make all && make install DESTDIR=/tmp/chkconfig
+
+  unitTests:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+    needs: buildcheck
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v2
+      - name: Clone beakerlib repository
+        run: git clone https://github.com/beakerlib/beakerlib.git
```

```suggestion
        run: git clone --depth=1 https://github.com/beakerlib/beakerlib.git
```

This should make the cloning slightly faster, since we don't need the full history.

---

Thank you.